src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+
+        final String Path = "fruitShop.csv";
```

Comment:
The path to the input file should be placed in the `src/main/resources` folder and referenced relatively.